games/modes/team.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Team mode
+ * Cassius - https://github.com/sirDonovan/Cassius
+ *
+ * This file contains code for a game mode (Team)
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+const name = 'Team';
+const id = Tools.toId(name);
+const data = {};
+
+data["Teams"] = Tools.data.teams;
+
+let TeamMode = function () {
+	this.name += ' ' + name;
+	this.id += id;
+	this.freeJoin = false;
+	this.playerList = [];
+	this.teamA = '';
+	this.teamB = '';
+	this.teamAplayers = [];
+	this.teamBplayers = [];
+	this.points = new Map();
+	this.maxPoints = 20;
+	this.onSignups = null;
+
+	this.onStart = () => {
+		if (this.playerCount < 2) {
```

Comment:
Yes, I would change <code>if (this.started) return;</code> to <code>if (this.started || (this.modeId === 'team' && this.playerCount < 2)) return;</code>.